examples/vanilla-js/vanilla-custom-side-menu/index.html

```
@@ -0,0 +1,14 @@
+<html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)